index.html

```diff
@@ -7,10 +7,43 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <div class="contener">
+        <div class="header">
```

Comment:
Instead of using the div as a header, have you thought about using <header> tag instead?